features/guidance/collapse.feature

```diff
@@ -143,37 +143,33 @@ Feature: Collapse
     Scenario: Partly Segregated Intersection, Two Segregated Roads
         Given the node map
             """
-              n   m
-
-
-
-
-
-              g   h
-
-
-            c   b   a
-            d   e   f
-
-
-              j   i
-
-
-
-
-
-              k   l
+               n m
+               | |
+               | |
+               | |
+               | |
+               | |
+               g h
+            c - b - a
+            d - e - f
+               j i
+               | |
+               | |
+               | |
+               | |
+               | |
+               k l
             """
 
         And the ways
-            | nodes | highway | name   | oneway |
-            | ab    | primary | first  | yes    |
-            | bc    | primary | first  | yes    |
-            | de    | primary | first  | yes    |
-            | ef    | primary | first  | yes    |
-            | be    | primary | first  | no     |
-            | ngbhm | primary | second | yes    |
-            | liejk | primary | second | yes    |
+            | nodes | highway | name   | oneway | lanes |
+            | ab    | primary | first  | yes    |       |
+            | bc    | primary | first  | yes    |       |
+            | de    | primary | first  | yes    |       |
+            | ef    | primary | first  | yes    |       |
+            | be    | primary | first  | no     |       |
+            | ngbhm | primary | second | yes    | 5     |
+            | liejk | primary | second | yes    | 5     |
```

Comment:
see above. 2.5 * 2 *3 = 15 meters so we got 5 meters to spare between the roads.